Extensions/quick_tags.js

```diff
@@ -354,8 +372,9 @@ XKit.extensions.quick_tags = new Object({
 
 		var user_tag_array = XKit.extensions.quick_tags.load_tag_prefs();
 		var m_user_tags = XKit.extensions.quick_tags.render_tags_from_array(user_tag_array, true);
+		var m_add_button = XKit.extensions.quick_tags.render_add_bundle_button(true);
```

Comment:
cpanel doesn't use render_tags_from_array

On Sat, Aug 8, 2015 at 8:12 AM ThePsionic <notifications@github.com> wrote:

> In Extensions/quick_tags.js
> <https://github.com/new-xkit/XKit/pull/173#discussion_r36578081>:
>
> > @@ -354,8 +372,9 @@ XKit.extensions.quick_tags = new Object({
> >
> >  		var user_tag_array = XKit.extensions.quick_tags.load_tag_prefs();
> >  		var m_user_tags = XKit.extensions.quick_tags.render_tags_from_array(user_tag_array, true);
> > +		var m_add_button = XKit.extensions.quick_tags.render_add_bundle_button(true);
>
> Nope, since we only want to add it in the 1-click postage bundle list, and
> not in the cpanel bundle list.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/173/files#r36578081>.
>